lib/fakeroot.py

```diff
@@ -48,19 +146,45 @@
    #
    # 1. CentOS seems to have only fakeroot, which is in EPEL, not the standard
    #    repos.
+   #
+   # 2. Enabling EPEL can have undesirable side effects, e.g. different
+   #    version of things in the base repo that breaks other things. Thus,
+   #    when we install EPEL, we don't enable it. Existing EPEL installations
+   #    are left alone.
```

Comment:
Thought I saw a new `--enable` line in the epel configuration of the centos7 image above. Is that for testing?

Reply:
@j-ogas see below around line 169. When epel-release is installed automatically via `--force` is it disabling the epel repo.